### PR TITLE
Fix any sequencer typo

### PR DESCRIPTION
**Pull request by Tomcli**
**Which issue is resolved by this Pull Request:** 
Related #499 

**Description of your changes:**
Spelling error on the word "sequencer", move the any sequencer module to the exterior layer to match with the platform configurations like gcp and onprem.

**Environment tested:**

* Python Version (use `python --version`):
* Tekton Version (use `tkn version`):
* Kubernetes Version (use `kubectl version`):
* OS (e.g. from `/etc/os-release`):

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

File: sdk/FEATURES.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ The `finally` syntax is supported since Tekton version `0.14.0`.
 
 ### Pipeline Loops
 
-PipelineLoops is a feature for running a component or a set of component tasks multiple times in a loop. Right now, Tekton supports loop pipeline/tasks via an implementation of [Tekton Custom Tasks](https://github.com/tektoncd/community/blob/master/teps/0002-custom-tasks.md) controller named as "PipelineLoop". Please refer to the examples [here](/tekton-catalog/pipeline-loops/examples) to understand more details about the usage of loops. 
+PipelineLoops is a feature for running a component or a set of component tasks multiple times in a loop. Right now, Tekton supports loop pipeline/tasks via an implementation of [Tekton Custom Tasks](https://github.com/tektoncd/community/blob/master/teps/0002-custom-tasks.md) controller named as "PipelineLoop". Please refer to the examples [here](/tekton-catalog/pipeline-loops/examples) to understand more details about the usage of loops.
```
Comment (animeshsingh) [inline sdk/FEATURES.md:131]:
```suggestion
PipelineLoops is a feature for running a component or a set of component tasks multiple times in a loop. Right now, Tekton supports loop pipeline/tasks via an implementation of [Tekton Custom Tasks Controller](https://github.com/tektoncd/community/blob/master/teps/0002-custom-tasks.md) named as "PipelineLoop". Please refer to the examples [here](/tekton-catalog/pipeline-loops/examples) to understand more details about the usage of loops.
```

**Review by ckadner — APPROVED** [review]

**Comment by Tomcli** [comment]
@pugangxa I moved the any_sequencer class to the exterior level inside the tekton.py. DSL directory is for configurations that work on any platform. Since any_sequencer is getting status from taskRun crd (and Argo doesn't have crd for each task status), it might make more sense to put it under a list of helper function for the tekton platform. Just like kfp has all the platform specific helper functions on the exterior level, not DSL. 
https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/onprem.py

**Review by ckadner — CHANGES_REQUESTED** [review]
Hi @Tomcli -- could we also rephrase the class comment in the AnySequencer  https://github.com/kubeflow/kfp-tekton/blob/7b96438944f44ec23f5bbc0dbd3155ce7ecbccd2/sdk/python/kfp_tekton/tekton.py#L23-L31

File: sdk/python/kfp_tekton/tekton.py [hunk 2]
```diff
@@ -20,16 +20,15 @@
 
 
 class AnySequencer(ContainerOp):
-    """Represents an op present that will proceed when any denpendent condition met
+    """A containerOp that will proceed when any dependent condition met
```
Comment (ckadner) [inline sdk/python/kfp_tekton/tekton.py:23]: maybe ..." _if any of the dependent containerOps completed successfully_"

**Review by ckadner — COMMENTED** [review]
thank you @Tomcli

File: sdk/python/kfp_tekton/tekton.py (hunk 2 above)
Comment (Tomcli) [reply sdk/python/kfp_tekton/tekton.py]: done

**Comment by fenglixa** [comment]
Thanks @Tomcli , The changes is looks good to me.
/lgtm

**Comment by pugangxa** [comment]
> @pugangxa I moved the any_sequencer class to the exterior level inside the tekton.py. DSL directory is for configurations that work on any platform. Since any_sequencer is getting status from taskRun crd (and Argo doesn't have crd for each task status), it might make more sense to put it under a list of helper functions for the tekton platform. Just like kfp has all the platform specific helper functions on the exterior level, not DSL.
> https://github.com/kubeflow/pipelines/blob/master/sdk/python/kfp/onprem.py

Thanks Tommy, agree we should follow the convention!

**Pull request merged**